prisma/sql/getAllocatorBiggestClientDistribution.sql

```diff
@@ -1,23 +1,14 @@
-with allocators_with_ratio as (
-    select
-        week,
-        allocator,
-        max(sum_of_allocations) / sum(sum_of_allocations) biggest_to_total_ratio
-    from client_allocator_distribution_weekly
-    group by
-        week,
-        allocator
-)
-select
-    week,
-    100 * ceil(biggest_to_total_ratio::float8 * 20) / 20 - 5 as "valueFromExclusive",
-    100 * ceil(biggest_to_total_ratio::float8 * 20) / 20 as "valueToInclusive",
-    count(*)::int as count
+with allocators_with_ratio as (select week                    as                        "week",
+                                      allocator               as                        "allocator",
+                                      max(sum_of_allocations) / sum(sum_of_allocations) biggest_to_total_ratio,
```

Comment:
```suggestion
                                      max(sum_of_allocations) / sum(sum_of_allocations) "biggestToTotalRatio,
```
to make it consistent? Applies to other aggregations as well.